README.md

```diff
@@ -62,6 +62,26 @@ Or in case of an error:
 }
 ```
 
+Verifying Signatures
+-------------
+
+We sign our HTTP requests to allow you to verify that they actually came from us. Let's use Signature middleware to verify webhooks.
```

Comment:
Maybe include a link to https://developers.messagebird.com/docs/verify-http-requests here as well :)

Reply:
✅